DashAI/back/database/models.py

```diff
@@ -43,21 +43,8 @@ class ModelInstance(Base):
     experiment_id: Mapped[int] = mapped_column(ForeignKey("experiment.id"))
     parameters: Mapped[JSON] = mapped_column(JSON)
     model_name: Mapped[str] = mapped_column(String)
-    run: Mapped["Run"] = relationship(back_populates="run", cascade="all, delete")
-
-
-class Run(Base):
-    __tablename__ = "run"
-    """
-    Table to store all the information about a specific run of a model.
-    """
-    id: Mapped[int] = mapped_column(primary_key=True)
-    model_id: Mapped[int] = mapped_column(ForeignKey("model_instance.id"))
     train_results: Mapped[JSON] = mapped_column(JSON)
     test_results: Mapped[JSON] = mapped_column(JSON)
     validation_restuls: Mapped[JSON] = mapped_column(JSON)
-    weights_path: Mapped[str] = mapped_column(String)
+    binary_path: Mapped[str] = mapped_column(String)
```

Comment:
Two things here:

1. I've seen a lot of the artifact concept (https://h2o.ai/wiki/artifacts/, https://machine-learning.paperspace.com/wiki/artifacts , etc.). Maybe it would be good to change the name to fit that standard?
```suggestion
    artifact_path: Mapped[str] = mapped_column(String)
```
2. Where would other training outputs such as interpretability graphs or roc curves graphs be stored?

Reply:
1. Name makes sense.
2. Thats a good question, i would leave those graphs as part of a key of the train_results/test_results/validation_results JSON's. I have an idea but thats part of the metric object which isnt implemented. The JSON should contain: {"roc_curve": path_to_curve ... } or maybe { "roc_curve" : base64_encoded_png ... } (but i dislike second option)